api/app/src/main/resources/application.properties

```
@@ -18,6 +18,11 @@ management.endpoints.web.exposure.include=health,prometheus
 spring.mvc.throw-exception-if-no-handler-found=true
 spring.web.resources.add-mappings=false
 server.error.include-stacktrace=never
+#task execution
+spring.task.execution.pool.core-size=${PACKIT_TASK_EXECUTOR_CORE_POOL_SIZE:4}
+spring.task.execution.pool.max-size=${PACKIT_TASK_EXECUTOR_MAX_POOL_SIZE:10}
+spring.task.execution.pool.queue-capacity=${PACKIT_TASK_EXECUTOR_QUEUE_CAPACITY:100}
+spring.task.execution.pool.keep-alive=${PACKIT_TASK_EXECUTOR_KEEP_ALIVE_SECONDS:60}
```

```suggestion
spring.task.execution.pool.core-size=4
spring.task.execution.pool.max-size=10
spring.task.execution.pool.queue-capacity=100
spring.task.execution.pool.keep-alive=60
```

I understand you're just following the existing pattern in this file, but there's really no need for us to define any of these environment variables. Spring Boot supports "externalized configuration", which means it will read properties from all kinds of sources and merge them before loading the application. This includes environment variables already. These values can be overridden by setting `SPRING_TASK_EXECUTION_POOL_CORESIZE` etc..

https://docs.spring.io/spring-boot/reference/features/external-config.html#features.external-config.typesafe-configuration-properties.relaxed-binding.environment-variables